.ibm/pipelines/resources/rhdh-operator/installation/rhdh-subscription.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: rhdh-operator
```

Comment:
@Omar-AlJaljuli Does this deploy the latest and new rhdh build on `main`?

Reply:
This installs the latest GA version of RHDH. I was told that only GA will work with IBM, since IBM is a self-hosted cluster.